source/classes/InteractionWrapper.js

```diff
@@ -135,9 +135,72 @@ class SelectWrapper extends InteractionWrapper {
 	 * @param {number} cooldownInMS
 	 * @param {(interaction: AnySelectMenuInteraction, args: string[], database: Sequelize, runMode: "prod" | "migration" | undefined) => void} executeFunction
 	 */
-	constructor(mainIdInput, cooldownInMS, executeFunction) {
+	constructor(mainIdInput, descriptionInput, defaultMemberPermission, isPremiumCommand, allowInDMsInput, cooldownInMS, executeFunction) {
+		super(mainIdInput, cooldownInMS, executeFunction);
+		this.premiumCommand = isPremiumCommand;
+		this.builder = new ContextMenuCommandBuilder()
+			.setName(mainIdInput)
+			.setDescription(descriptionInput)
+			.setDMPermission(allowInDMsInput)
+			.setType();
+		if (defaultMemberPermission) {
+			this.builder.setDefaultMemberPermissions(defaultMemberPermission);
+		}
+	}
+};
```

Comment:
Were these changes to `SelectWrapper` intended? The empty `.setType()` and other interface changes seem like they might not function properly.

Reply:
They were not, and I have changes ready to fix this soon.